sdk/rust/oak/src/grpc/server.rs

```diff
@@ -34,7 +34,10 @@ pub fn init(address: &str) -> Result<Receiver<Invocation>, OakStatus> {
     let (invocation_sender, invocation_receiver) =
         crate::io::channel_create::<Invocation>("gRPC invocation", &Label::public_untrusted())
             .expect("Couldn't create gRPC invocation channel");
-    init_with_sender(address, invocation_sender)?;
+    init_with_sender(address, invocation_sender).map_err(|e| {
```

Comment:
Personally I would not attempt to manually fix some of the leaks like this, rather try to figure out if it can be indeed solved more fundamentally as part of #1686 . Anyway, up to you.

Reply:
I'd like to leave the manual fixes in place, so that the stricter leak tests can be part of the CI runs.  That way any implementation of #1686 doesn't have to remember to come back and enable leak tests.